crates/database/src/lib.rs

```diff
@@ -0,0 +1,32 @@
+use sqlx::PgExecutor;
+
+// The names of all tables we use in the db.
+pub const ALL_TABLES: &[&str] = &[
+    "orders",
+    "trades",
+    "invalidations",
+    "quotes",
+    "settlements",
+    "presignature_events",
+    "order_quotes",
+];
+
+/// Delete all data in the database. Only used by tests.
+pub async fn clear(ex: impl PgExecutor<'_> + Copy) -> sqlx::Result<()> {
```

Comment:
So scary this exists....

Reply:
One idea - can we add a
```rust
#[cfg(not(any(test, debug_assertions)))]
compile_error!("this method can only be used in debug or test configurations");
```

Reply:
The downside, is we can't do `cargo test -p other_crate --release` if it uses the `clear` function.

Reply:
I'm not super against it but I feel it isn't necessary.

Reply:
I am quite scared 😛 TBH - I've seen `dbg!` sneak in already here and there.

Reply:
Renamed to `clear_DANGER`

Reply:
Ah sorry - meant to comment on it today.

I agree with your points, and appreciate you meeting me half-way with the rename of the method. 🙏